lib/ansible/playbook/play.py

```diff
@@ -130,8 +130,8 @@ def preprocess_data(self, ds):
             # this should never happen, but error out with a helpful message
             # to the user if it does...
             if 'remote_user' in ds:
-                raise AnsibleParserError("both 'user' and 'remote_user' are set for %s. "
-                                         "The use of 'user' is deprecated, and should be removed" % self.get_name(), obj=ds)
```

Comment:
This was calling `self.get_name()` but `load_data()` doesn't set that until after it calls this, so `self.name` was always empty.

Reply:
I'm actually working on a fix for that in #74147